Makefile

```
@@ -7,6 +7,7 @@ test: checkstyle test-unit
 .PHONY: test-unit
 test-unit: test-more-bash
 	prove -r test/
+	py.test tests/trigger-bisect-jobs.py
```

Make that

```suggestion
	py.test
```

and rename tests/trigger-bisect-jobs.py to tests/test_trigger_bisect_jobs.py

---

done